posts/aws-summit-2018nyc.md

```diff
@@ -0,0 +1,36 @@
+---
+authors:
+- Randeep Walia
+categories:
+- AWS
+date: 2018-08-13T06:25:27.000Z
+title: "AWS Summit NYC 2018"
+---
+The glass ceiling of the Javits Center cracked, literally, on a rainy day in July. As water poured from the roof to the floor several stories below, one couldn't be blamed for attributing it to the collective energy in the room radiating out from the hundreds of software engineers and architects that filled the space. The AWS Summit had made its way to New York City and I was in attendance to witness it.
+
+AWS Summit is a whirlwind- even the most experienced Amazon Web Services developer would probably be impressed by some new revelation during this day of technology in the cloud. The ecosystem is huge and the capabilities of its toolset can be jaw dropping. Over the course of the day I attended workshops, keynote speeches, sales pitches and sat down here to try and separate some of they hype from the reality. Here is a summary of some things I learned and products being pushed out of AWS' diverse offerings.
+
+## Databases 
+Amazon is pushing its managed database services hard. There were recurring workshops on [Aurora](https://aws.amazon.com/rds/aurora/), Amazon's fully managed solution intended to replace your MySQL or PostgreSQL. This includes the release of Aurora Serverless, which can autoscale up or down depending on the needs of a severless application. If you are not looking to go all-in on AWS there are other options as well. MongoDB was present pitching [Atlas](https://www.mongodb.com/cloud/atlas), their fully-managed NoSQL database in the cloud for AWS, GCP, and Azure. 
+
+Measuring the true cost of a fully-managed database vs a cloud hosted solution vs an on-premise solution is not a trivial task and a lot of factors have to be considered. But for business solutions that don't want to worry about the headache of patching, upgrading, scaling and fine-tuning a solution, a managed solution offers many nice features.
+
+## Serverless Architectures
+Full stack developers and Solution Architects have a myriad of options at their disposal. In addition to all the backend storage options mentioned above, [serverless architecture patterns](https://blog.ippon.tech/going-serverless/) are transforming the way Internet solutions can be developed in the cloud. 
+
+Applications that are data intensive can use Kinesis Streams to ingest and process massive amounts of data (a presentation from Fortnite, one of the world's most popular mobile games, revealed that the developers are ingesting and processing petabytes of data each day). 
```

Comment:
You could say something like "a presentation by Epic Games, creator of Fortnite".  I only know this because I have a former coworker who works there and I always wonder if he's a gazillionaire now.